src/browser/base/content/zen-styles/zen-tabs/vertical-tabs.css

```diff
@@ -300,7 +299,6 @@
               box-shadow: 0 0 1px 1px rgba(0, 0, 0, 0.15) !important;
             }
           }
-          margin-inline: var(--tab-block-margin);
```

Comment:
This is necessary for the shadows to appear

Reply:
Shadows are still there. The issue with this margin being here is the inconsistency.
See:
![PowerToys CropAndLock_v7CKalk4EL](https://github.com/user-attachments/assets/b3298a34-6410-4c6a-803a-127346948815)

I can change it so all tabs (collapsed & expanded) have this margin applied.